mlir/lib/Dialect/WebAssembly/IR/WebAssemblyOps.cpp

```diff
@@ -89,6 +89,56 @@ Block *BlockReturnOp::getTarget() {
       ->getSuccessor(0);
 }
 
+//===----------------------------------------------------------------------===//
+// ExtendFromLowBytesSOp
+//===----------------------------------------------------------------------===//
+
+ParseResult ExtendFromLowBytesSOp::parse(::mlir::OpAsmParser &parser,
+                                         ::mlir::OperationState &result) {
+  OpAsmParser::UnresolvedOperand operand;
+  uint64_t nBits;
+  auto parseRes = parser.parseInteger(nBits);
+  parseRes = parser.parseKeyword("low");
```

Comment:
Nit, feel free to ignore if you disagree: I'm not super fond of having very verbose operations like that - I feel like we could convey the meaning of this operation with something like `extend X %op` or `extend Xb %op`

Reply:
I don't have a strong opinion on it, but I feel in this case we would need to change the mnemonic because extend X op doesn't help to understand what this X is about and I still think the destination of text assembly format is Homan reader (I would expect these keywords have no existence when using the binary encoding of the dialect)